content/events/hacktoberfest.adoc

```diff
@@ -133,8 +133,26 @@ Web and CLI interfaces, and the codebase:
 
   link:https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#adding-a-logo[Adding a logo]
 
+
+| link:https://github.com/jenkinsci/helm-charts[Jenkins Helm Chart]
+| Kubernetes, Helm, Documentation
+a| `stable/jenkins` helm chart was recently been moved to link:https://github.com/jenkinsci/helm-charts[jenkinsci/helm-charts].
+  The goal is to release a 3.0.0 version of the chart, which removes old or deprecated configuration options, simplifies the Chart and makes it easier to use.
+  There are many smaller and bigger task which could be tackled. Every contribution counts. Here are some ideas:
```

Comment:
"There are many smaller and bigger task which could be tackled. Every contribution counts." this part could be removed, we mention it on the page for all projects